### ISSUE TITLE
`<section>` should have an `aria-label` or a `aria-labelledby` attribute

### ISSUE THREAD

**Issue by vtlanglois** [vtlanglois/vtlanglois.github.io#40]
For each `<section>` element, as it has a `region` role [^1], should have an accessible name. 

## Approach
- If a section has an heading that easily describes the section, use `aria-labelledby` with an `id` to the section's `SectionHeader`
- Else, give the `<section>` an `aria-label` with text that describes the section

[^1]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role#description
